### ISSUE TITLE
Characterizing the Impacts of Semi-supervised Learning for Weak Supervision	

### ISSUE THREAD

**Issue by reyllama** [reyllama/paper-reviews#228]
# TL; DR

- Organize the design space of Weakly Supervised / Semi-Supervised Learning frameworks.

# Motivation

- Weakly supervised learning and semi-supervised learning have much in common, but they have not been studied in a coupled context.

# Formulation

![image](https://github.com/reyllama/paper-reviews/assets/46308407/88ffe353-5fdd-472d-bd48-d880b7c7efc4)

- Thresholding: *what to treat as unlabeled*
- Semi-sup technique: *what to do with the unlabeled*
- Relabeling: *how will update the data labels dynamically*

![image](https://github.com/reyllama/paper-reviews/assets/46308407/ffab206b-872b-4ed1-8006-8dfde5a8ce72)

# Evaluation

![image](https://github.com/reyllama/paper-reviews/assets/46308407/1c19ff4a-13e8-469e-b831-ec3f69e92cbc)

![image](https://github.com/reyllama/paper-reviews/assets/46308407/d2569064-cda3-4bb8-b06a-16c9857de46c)

![image](https://github.com/reyllama/paper-reviews/assets/46308407/027c5f15-2b6b-4ae1-b68a-26e9fc1cb111)

![image](https://github.com/reyllama/paper-reviews/assets/46308407/58d7e716-b649-4275-b6fa-921a63de1942)

# Inspiration

- WSL and SSL (semi) have much in common, and the effectiveness of adding SSL on top of WSL varies w.r.t. the dataset.
- Of course, if there are more unlabeled samples in the dataset, employing semi-supervised learning to utilize those samples boost the performance.
